docs/guides/storage.md

```diff
@@ -116,6 +116,9 @@ Note that the original virtual environment is still installed in `$SCRATCH/sqfs-
 
 A benefit of this approach is that the squashfs file can be copied to a location that is not subject to the Scratch cleaning policy.
 
+??? warning
+    Most likely, your virtual environment is not relocatable as it contains symlinks to paths in the original location on the filesystem. Therefore, you need to mount the image in the exact same location where you created the virtual environment.
```

Comment:
`Virtual environments are usually not relocatable because they contain symlinks to locations inside the virtual environment.`?